docs/modules/m1/pages/walkthrough.adoc

```diff
@@ -1,7 +1,100 @@
-# Title
-# Red Hat Summit 2023
+# Red Hat Summit 2023: Vote App Lab Overview - Walkthrough
 
-## LAB WALKTHROUGH
+In this section, we'll take a look at the architecture that this lab will be using.
 
+.Architecture
+image::https://github.com/cedricclyburn/vote-app-gitops/raw/main/images/demo-architecture.png[Architecture]
 
-### Module One
+NOTE: Everything has already been pre-installed on your cluster, so there's no need to do anything here. This is just for reference.
+
+## Lab Architecture
+
+To run this lab, we're using an OpenShift 4.12 cluster. This cluster has been pre-configured with the following:
+
+### Operators
+
+#### OpenShift Pipelines 
+*OpenShift Pipelines* is a cloud-native, continuous integration and delivery (CI/CD) solution for building pipelines using Tekton. Tekton is a flexible, Kubernetes-native, open-source CI/CD framework that enables automating deployments across multiple platforms (Kubernetes, serverless, VMs, etc) by abstracting away the underlying implementation details.
+
+Follow these instructions in order to install OpenShift Pipelines on
+OpenShift via the OperatorHub:
+
+- From the left-side menu under *Administrator* perspective, go to
+*Operators*-> *OperatorHub*. In the search box, search for _pipelines_,
+then click to *OpenShift Pipelines Operator*:
+
+.OperatorHub
+image::../assets/images/prerequisites_operatorhub.png[OperatorHub]
```

Comment:
@blues-man here's how I have it importing, but it's not displaying when I run the container...

Reply:
Sorry for the late reply. It works for me after a local test, so I'll assume it has been fixed. Let me know otherwise. @cedricclyburn

Reply:
My bad. I see there were many other images with the same issue. The problem is on the image path. I fixed one of them as an example, check for my commit on your (this) branch.